pkg/cmd/image/convert.go

```diff
@@ -75,6 +75,12 @@ func Convert(ctx context.Context, client *containerd.Client, srcRawRef, targetRa
 	}
 	convertOpts = append(convertOpts, converter.WithPlatform(platMC))
 
+	// Ensure all the layers are here: https://github.com/containerd/nerdctl/issues/3425
+	err = EnsureAllContent(ctx, client, srcRawRef, options.GOptions)
+	if err != nil {
+		return err
```

Comment:
Convert and Save DO hard error, since the operation cannot complete if there is missing content.